mozci/push.py

```diff
@@ -471,9 +515,25 @@ def get_regressions(self, runnable_type):
         """
         regressions = {}
 
+        # If the push was not backed-out and was not "bustage fixed", it can't
+        # have caused regressions.
+        if not self.backedout and not self.bustage_fixed_by:
+            return regressions
+
+        # Any failure that comes after this push has been backed-out or bustage fixed,
+        # can't be blamed on this push.
+        other = self.child
+        for child_count_cutoff in range(MAX_DEPTH + 1):
+            if self.backedoutby in other.revs or self.bustage_fixed_by in other.revs:
```

Comment:
Maybe we should have a `fixedby` property that combines both the `backedoutby` and `bustage_fixed_by` properties into one. We could still differentiate backouts via `push.backedout` if we needed to.

Reply:
I think we can do this as part of #125.